### PR TITLE
create simple documentation

### PR DESCRIPTION
**Pull request by MichalMilewicz**
Fill up README.md file describing setup process, how to test, how to deploy and start in development env, and attach it to Sphinx docs engine.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+   make serve
+
+**You will get application working on localhost:3000
+Debugger in on**
+
+HOW TO RUN TEST
+---------------
+
+Enter to Tungsten Dashboard directory
```
Comment (pawelzny) [inline README.rst]: This section should look similar to one before.
```rst
Run tests
---------

.. code:: console

    $ source .venv/bin/activate
    $ make test && make lint
```

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+   cd [dir_with_file]
+
+Run tests
+
+.. code::
+
+   make test
+
+**All tests You can find in /tests**
```
Comment (pawelzny) [inline README.rst]: IMO redundant information. You can easily spot tests directory by looking at source tree.

File: docs/conf.py [hunk 3]
```diff
@@ -15,6 +15,7 @@
 import os
 import sys
 
+
```
Comment (pawelzny) [inline docs/conf.py]: This change does nothing. Please do not make such changes unless linter complains about it.

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+Awesome CI Dashboard is open source client interface for Zuul Gating system
+
+More information about Zuul Gating system you can find at
+
+https://zuul-ci.org/docs/zuul/
+
+Project actual status is development stage, work only in development environment
+
+Tungesten Dasboard use APACHE 2.0 LICENSE, for more info check LICENSE
```
Comment (kawazaki0) [inline README.rst]: use -> uses

File: README.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+   git clone git@github.com:tungstenfabric/tungsten-ci-dashboard.git
+   cd tungsten-ci-dashboard/
+
+Set venv with dependencies for project:
+
+.. code::
+
+   python3.6 -m venv .venv
+   source .venv/bin/activate
+   pip install setuptools wheel
```
Comment (kawazaki0) [inline README.rst]: @pawelzny if this command is necessary in setup virtualenv, maybe we can add it to Makefile?

File: README.rst [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+
+
+RUN Awesome CI Dashboard
+--------------------------------
+
+
+.. code:: console
+
+    $ source .venv/bin/activate
```
Comment (kawazaki0) [inline README.rst]: not consistent with other shell command examples

File: README.rst [hunk 7]
```diff
@@ -1,26 +1,19 @@
...
+
+More information about Zuul Gating system you can find at
 
-https://github.com/tungstenfabric/website
+https://zuul-ci.org/docs/zuul/
 
 Project actual status is development stage, work only in development environment
 
-Tungesten Dasboard use APACHE LICENSE, for more info check LICENSE
+Tungesten Dasboard use APACHE 2.0 LICENSE, for more info check LICENSE
```
Comment (magickris93) [inline README.rst]: Tungesten -> Tungsten

Comment (pawelzny) [reply README.rst]: Tungsten -> ACID

File: README.rst [hunk 8]
```diff
@@ -0,0 +1,59 @@
+TUNGSTEN DASHBOARD
```
Comment (kam193) [inline README.rst]: Awesome CI Dashboard

File: README.rst [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+==================
+
+Awesome CI Dashboard is open source client interface for Zuul Gating system
```
Comment (kam193) [inline README.rst]: Better: ``user interface``

Comment (kawazaki0) [inline README.rst]: Full project name and abbreviation at the beginning would be nice. `Awesome CI Dashboard (ACID)`

File: README.rst [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+https://zuul-ci.org/docs/zuul/
+
+Project actual status is development stage, work only in development environment
+
+Tungesten Dasboard use APACHE 2.0 LICENSE, for more info check LICENSE
+
+HOW TO INSTALL
+--------------
+
+Install/check installation of python 3.6/venv:
```
Comment (pawelzny) [inline README.rst]: IMO this is confusing. Because you don't want to use `apt install` to check if binaries exist.
I would change it to 

> ACID require Python 3.6.x. Please make sure you have Python 3.6 version in your OS. If not install it before running project.

File: README.rst [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+
+.. code:: console
+
+   $ sudo apt install python3.6 python3.6-venv
+
+Set venv with dependencies for project:
```
Comment (pawelzny) [inline README.rst]: `Set` implice that you already have venv and you want to change it's value or state.
IMO it should be 

> Create new virtual environment for ACID project, unless you don't care about PIP packages collision.

File: README.rst [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+
+Run vagrant for remote services like Zuul and Gerrit for development purposes.
+
+.. code:: console
+
+    $ make dev-run
+
+
+RUN Awesome CI Dashboard
+--------------------------------
```
Comment (pawelzny) [inline README.rst]: Lines under heading must be equal to length of heading.
For example if heading has 5 letters then it should be 5 hyphens.
```rst
Run ACID
--------
```

File: README.rst [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+
+.. code:: console
+
+    $ source .venv/bin/activate
+    $ make serve
+
+Server will listen on http://127.0.0.1:3000/
+
+RUN TESTS
+---------------
```
Comment (pawelzny) [inline README.rst]: To many heading characters.

File: README.rst [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+RUN Awesome CI Dashboard
+--------------------------------
+
+
+.. code:: console
+
+    $ source .venv/bin/activate
+    $ make serve
+
+Server will listen on http://127.0.0.1:3000/
```
Comment (pawelzny) [inline README.rst]: Make this URL clickable
```rst
[http://127.0.0.1:3000/](http://127.0.0.1:3000/)
```

Comment (MichalMilewicz) [reply README.rst]: Didn't make changes - this is clickable after running Sphinx

File: README.rst [hunk 15]
```diff
@@ -1,26 +1,24 @@
...
+Awesome CI Dashboard is open source user client interface for Zuul Gating system
 
 More information about Zuul Gating system you can find at
 
 https://zuul-ci.org/docs/zuul/
 
 Project actual status is development stage, work only in development environment
 
-Tungesten Dasboard use APACHE 2.0 LICENSE, for more info check LICENSE
+Tungesten Dasboard uses APACHE 2.0 LICENSE, for more info check LICENSE
```
Comment (pawelzny) [inline README.rst]: Change `Tungesten Dashboard` to `Awesome CI Dashboard` please.

File: README.rst [hunk 16]
```diff
@@ -0,0 +1,57 @@
+AWESOME CI DASHBOARD (ACID)
+===========================
+
+Awesome CI Dashboard is open source user client interface for Zuul Gating system
```
Comment (magickris93) [inline README.rst]: Please add 'an' before open. Also there is no client here, so ACID is just a user interface (instead of user client interface).

File: README.rst [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+
+    $ source .venv/bin/activate
+    $ make serve
+
+Server will listen on http://127.0.0.1:3000/
+
+RUN TESTS
+---------
+
+Enter to Tungsten Dashboard directory
```
Comment (magickris93) [inline README.rst]: Tungsten Dashboard -> ACID. Also remove 'to'.

File: README.rst [hunk 18]
```diff
@@ -0,0 +1,57 @@
+AWESOME CI DASHBOARD (ACID)
+===========================
+
+Awesome CI Dashboard is open source user client interface for Zuul Gating system
+
+More information about Zuul Gating system you can find at
+
+https://zuul-ci.org/docs/zuul/
+
+Project actual status is development stage, work only in development environment
```
Comment (magickris93) [inline README.rst]: Maybe use current instead of actual. They have similar meanings, but are used in different contexts.

I would also rephrase it to something like this: 

`Project is currently in development stage and only works in a development environment.`

File: README.rst [hunk 19]
```diff
@@ -0,0 +1,57 @@
+AWESOME CI DASHBOARD (ACID)
+===========================
+
+Awesome CI Dashboard is open source user client interface for Zuul Gating system
+
+More information about Zuul Gating system you can find at
```
Comment (magickris93) [inline README.rst]: Please add a colon at the end of the line.

File: README.rst [hunk 20]
```diff
@@ -0,0 +1,57 @@
...
+
+https://zuul-ci.org/docs/zuul/
+
+Project actual status is development stage, work only in development environment
+
+Awesome CI Dashboard uses APACHE 2.0 LICENSE, for more info check LICENSE
```
Comment (magickris93) [inline README.rst]: Please add a full stop at the end of the sentence.

File: README.rst (hunk 14 above)
Comment (pawelzny) [reply README.rst]: OK, you are right. Leave it as it is.

File: README.rst [hunk 21]
```diff
@@ -0,0 +1,57 @@
...
+https://zuul-ci.org/docs/zuul/
+
+Project is currently in development stage and only works in a development environment.
+
+Awesome CI Dashboard uses APACHE 2.0 LICENSE, for more info check LICENSE.
+
+HOW TO INSTALL
+--------------
+
+ACID require Python 3.6.x. and Python3.6-venv  Please make sure you have Python 3.6 and Python 3.6 version in your OS.
```
Comment (kawazaki0) [inline README.rst]: `3.6.x.` -> `3.6.x` (without dot)
